validation/src/test/scala/hmda/validation/rules/lar/validity/BadValueUtils.scala

```diff
@@ -12,6 +12,11 @@ trait BadValueUtils extends LarGenerators {
     Gen.oneOf(belowRange, aboveRange)
   }
 
+  def intOtherThan(x: Any): Gen[Int] = x match {
+    case x: Int => Gen.choose(Int.MinValue, Int.MaxValue).filter(_ != x)
+    case x: List[Int] => Gen.choose(Int.MinValue, Int.MaxValue).filter(x.contains(_))
```

Comment:
Should this be a `filterNot`? It looks like this will only produce values in the list, instead of excluding values in the list.